### PR TITLE
Integrate identify snap

### PR DESCRIPTION
**Pull request by himalayan-dev**
## Description

This pull request changes the following:

- Integrate the identify snap onto the hashgraph repo

### CODE REVIEW

**Review by kpachhai — DISMISSED** [review]

**Review by kpachhai — DISMISSED** [review]

File: packages/hedera-identify-snap/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Hedera Wallet Snap for MetaMask
```
Comment (Nana-EC) [inline packages/hedera-identify-snap/README.md:1]: Contents of this file are incorrect and need to match the identify-snap

File: packages/hedera-identify-snap/packages/site/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Example Site to interact with Identity Snap
+
```
Comment (Nana-EC) [inline packages/hedera-identify-snap/packages/site/README.md:2]: Would be useful to have a paragraph highlighting the website functionality and a section for technologies utilized

File: packages/hedera-identify-snap/packages/site/gatsby-ssr.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*-
+ *
+ * Hedera Wallet Snap
```
Comment (Nana-EC) [inline packages/hedera-identify-snap/packages/site/gatsby-ssr.tsx:3]: Update all license headers to be correct for this package
```suggestion
 * Hedera Identify Snap
```

File: packages/hedera-identify-snap/packages/site/src/components/base/Buttons.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
+import { ComponentProps } from 'react';
```
Comment (Nana-EC) [inline packages/hedera-identify-snap/packages/site/src/components/base/Buttons.tsx:1]: tsx files are missing header licenses

File: packages/hedera-identify-snap/packages/snap/src/hedera/client/index.ts [hunk 5]
```diff
@@ -0,0 +1,117 @@
+import {
```
Comment (Nana-EC) [inline packages/hedera-identify-snap/packages/snap/src/hedera/client/index.ts:1]: Missing license headers in ts files

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Big step 🙌🏾 , let's add some missing repo items though.

**Review by kpachhai — APPROVED** [review]

**Review by kpachhai — DISMISSED** [review]

**Review by kpachhai — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**